automation/run-tests.sh

```diff
@@ -343,6 +350,7 @@ if [ -z ${RUN_BAREMETAL} ];then
     container_pre_test_setup
 else
     CONTAINER_WORKSPACE="."
+    run_customize_command
```

Comment:
This seems to change running the customize command only on non-container machines instead of only in containers.

Btw. IMHO the if statement could be more clean if you avoid negations:

```
if [ -n "${USE_CONTAINER}" ]
then

...

else
...
fi
```

would make it more clear that the first branch is for containers and the "else" branch for non-containers. Another alternative would be

```
if [ -n "${USE_MACHINE}" ]
    # do non container stuff
else
   # do container stuff
fi
```

But from our meeting discussion I remember that the distinction was "container" vs. "non container", therefore using "container" in the variable name makes more sense to me.

Reply:
I removed the negation and it is more clean now. Thanks!